girder/utility/install.py

```diff
@@ -175,6 +174,12 @@ def main():
     plugin.add_argument('-s', '--symlink', action='store_true',
                         help='Install by symlinking to the plugin directory.')
 
+    plugin.add_argument('--skip-web', action='store_true',
+                        help='Skip the step of running the web client build.')
+
+    plugin.add_argument('--skip-reqs', action='store_true',
+                        help='Skip the step of pip installing the requirements.txt file.')
+
```

Comment:
Nit: the options could be more self-explanatory if written as `--skip-web-client` and `--skip-requirements`.